src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -750,3 +752,45 @@ def get_local_connections_from_executable(executable):
             # ignore when connection not found since it can be configured with env var.
             raise Exception(f"Connection {n!r} required for flow {executable.name!r} is not found.")
     return result
+
+
+def get_current_env_name():
+    # Check if it's a Conda environment
+    conda_env = os.environ.get("CONDA_DEFAULT_ENV")
+    if conda_env:
+        return conda_env
+
+    # Check if it's a virtualenv environment
+    virtual_env_path = os.environ.get("VIRTUAL_ENV")
+    if virtual_env_path:
+        return os.path.basename(virtual_env_path)
+
+    # Check if Python executable is being called directly
+    # TODO: This might be hack, seek better way
+    python_exec_path = sys.executable
+    if python_exec_path:
+        env_path_parts = python_exec_path.split("envs")[1]
+        return env_path_parts.split(os.sep)[1]
+
+    # If it's neither Conda nor virtualenv, return 'default'
+    return "default"
+
+
+def refresh_connections_dir(connection_spec_files, connection_template_yamls):
```

Comment:
When do we use this function? didn't see reference of it.

Reply:
Removed temporialy for this pr